lib/components/molecules/Table/Table.story.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import defaultConfig from './Table.mock';
+import Table, { TableVanilla } from '.';
+
+storiesOf('Molecules', module)
+  .addParameters({ jest: ['Table', 'TableVanilla'] })
+
+  .add('Table Knobs', () => (
```

Comment:
Where is a test case and sample for table being able to hold existing elements in cell?